include/inexor/vulkan-renderer/wrapper/window.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include <vulkan/vulkan.h>
+
+#include <string>
```

Comment:
`#include <cstdint>` should probably be added.

Reply:
Nope, we need `std::string`.